updating links inside framework folder to https (#15460)
dotnet/docs: docs/framework/data/adonet/downloading-sample-databases-linq-to-dataset.md

  
1. Open Internet Explorer.  
  

> 2. Go to the [SQL Server 2005 Samples and Sample Databases](https://www.microsoft.com/downloads/details.aspx?FamilyId=E719ECF7-9F46-4312-AF89-6AD8702E4E6E&displaylang=en) Web site.  

Reviewer: ```suggestion
2. Go to the [SQL Server 2005 Samples and Sample Databases](https://www.microsoft.com/download/details.aspx?id=10679) Web site.
```

> 1. Start Internet Explorer.  

Reviewer: Seems like an outdated article ?
@mairaw, Does this suggestion seem to be good ?

```suggestion
1. Start your favorite browser, such as Microsoft Edge.
```

> 2. Go to the  [Microsoft SQL Server 2005 Express Edition](https://www.microsoft.com/downloads/details.aspx?familyid=220549B5-0B07-4448-8848-DCC397514B41&displaylang=en) download page.  

Reviewer: ```suggestion
2. Go to the  [Microsoft SQL Server 2005 Express Edition](https://www.microsoft.com/download/details.aspx?id=21844) download page.
```
